apps/website/src/pages/docs/alert.mdx

```diff
@@ -17,65 +17,64 @@ import * as AllExamples from 'examples';
+### Composition API
 
-### Informational
+You can build Alerts with sub-components that are fully customizable.
 
-Default style when there is no defined status for the alert. Inform users about events that they should be aware of, but that are not disruptive to their work.
+```tsx
+<Alert.Wrapper type='positive'>
+  <Alert.Icon />
+  <Alert.Message>
+    This is an alert
+    <Alert.Action onClick={() => console.log('Clicked more info!')}>Learn more</Alert.Action>
+  </Alert.Message>
+  <Alert.CloseButton onClick={() => console.log('CLOSED')} />
+</Alert.Wrapper>
+```
 
-<LiveExample src='Alert.informational.tsx'>
-  <AllExamples.AlertInformationalExample client:load />
-</LiveExample>
-
-### Positive
-
-The positive alert is used to inform the user of a positive status.
-
-<LiveExample src='Alert.positive.tsx'>
-  <AllExamples.AlertPositiveExample client:load />
-</LiveExample>
-
-### Warning
+The type of alert can be specified by utilizing the `type` prop inside `<Alert.Wrapper>`. There are four distinct types of alerts that can be chosen from:
 
-The warning alert is used to inform the user of something that is not currently disruptive to their work, but may be soon or eventually.
+- positive : indication of a task or action that has been successfully completed.
+- warning : requires attention but do not result in any immediate disruption.
+- negative : signify system outages or failures.
+- informational : intended to provide users with updates on non-disruptive tasks or work.
 
-<LiveExample src='Alert.warning.tsx'>
-  <AllExamples.AlertWarningExample client:load />
+<LiveExample src='Alert.types.tsx'>
+  <AllExamples.AlertTypesExample client:load />
 </LiveExample>
 
-### Negative
+### Legacy API
 
-Use this only when the system is broken. Leave the technical details behind and explain the problem to the user in an easily understood tone. Mention when the problem might get solved, if it applies.
+Alternatively, you can use `Alert` directly, without any subcomponents. Although this pattern can suffice for simple cases, we strongly recommend using composition API for more flexibility. Some of the legacy props like `clickableText`/`clickableTextProps` may be removed in the future, as they encourage bad patterns.
 
-<LiveExample src='Alert.negative.tsx'>
-  <AllExamples.AlertNegativeExample client:load />
-</LiveExample>
-
-### Multilines
-
-All alerts may support multiple lines if necessary. We recommend keeping it brief and keep the message as concise as possible.
+The following code is equivalent to the [composition example](#composition-api) above:
 
-## Placement
+```tsx
+<Alert
+  type='positive'
+  clickableText='More Info.'
+  onClose={() => console.log('Close!')}
```

Comment:
Can we make the Composition and Legacy examples fully equivalent? This one says "Close!" while the other one says "CLOSED"